src/core_ocean/Registry.xml

```diff
@@ -2292,12 +2292,21 @@
 		<var name="xtime" type="text" dimensions="Time" units="unitless"
 			 description="model time, with format 'YYYY-MM-DD_HH:MM:SS'"
 		/>
+		<var name="xtime_start" type="text" dimensions="Time" units="unitless"
+			 description="model time, with format 'YYYY-MM-DD_HH:MM:SS'"
+		/>
+		<var name="xtime_end" type="text" dimensions="Time" units="unitless"
+			 description="model time, with format 'YYYY-MM-DD_HH:MM:SS'"
+		/>
```

Comment:
@wenshanw, I don't think we want these as general variables.  They already exist in the time-series stats analysis member and I don't think they make sense anywhere else.  All other output is a snapshot so there is no start and end time.

Reply:
Yes, I realized that and removed them in the last commit.

Reply:
Okay, sounds good.  Just happened to see that and figured it might save you some time...